Update/omot_008_choice2.txt

```diff
@@ -106,7 +106,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6972c1>梨花</color>", NULL, "<color=#6972c1>Rika</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 5, "ps3/s14/05/210500460", 256, TRUE);
 	OutputLine(NULL, "「大局的に見れば…なんて、勝手な言い分だと思います。",
-			NULL, "\"I think it would be selfish... for me to say yes, if you look at it from the big picture.", Line_WaitForInput);
+			NULL, "\"I know it would be selfish of me... to make a excuse like \"looking it from the big picture.\"", Line_WaitForInput);
```

Comment:
```suggestion
			NULL, "\"I know it would be selfish of me... to make an excuse like \"looking at it from the big picture.\"", Line_WaitForInput);
```